litmus-portal/frontend/src/components/CustomStepper/index.tsx

```diff
@@ -157,8 +157,9 @@ const CustomStepper = () => {
               <ButtonOutline
                 isDisabled={activeStep === 0}
                 handleClick={handleBack}
-                value="Back"
-              />
+              >
+                <Typography>Back</Typography>
```

Comment:
The ButtonOutline already uses a typography to display the value, why wrap around another one here.

Reply:
@gdsoumya The children property of ButtonOutline component is taking a JSX.Element and can be reused to wrap a whole child as well as individual properties, that's why it wasn't taking only the Text. It needed to wrapped with at least a JSX Element.

Reply:
So we can put it in a <></> instead of another typography component?

Reply:
Fixed it